datasource/tests/tests.py

```diff
@@ -302,3 +306,43 @@ def test_recommend_brands(self):
         self.assertTrue(
             SuggestedAssociation.objects.get(brand=brand2, datasource=usnic5).certainty > 0
         )
+
+
+class FixtureLoadingTestCase(TestCase):
+    def setUp(self):
+        """
+        This method will generate the sample_initial.json file which contains exactly one entry
+        for each module. The size of this file will be very small as compared to the original
+        initial.json
+        """
+
+        file = open("fixtures/initial/initial.json", "r")
+        json_file = json.load(file)
+
+        sample_initial_json = list({data["model"]: data for data in json_file}.values())
+
+        for data in sample_initial_json:
```

Comment:
This nested for loop smells. I suspect it's bad practice to have a for loop over a list, and then another one looping over the keys and values within the list.

Reply:
Oh okay, Sure Albert. I will take a note of it and try to avoid these things in next ticket.